### PR TITLE
New injured mood

### PR DESCRIPTION
**Pull request by Rai99**

### CODE REVIEW

**Comment by kubastory9** [comment]
That's pretty good.

**Comment by Rai99** [comment]
Thanks!

I'm glad to know somebody likes my work.

File: Monika After Story/game/script-moods.rpy [hunk 1]
```diff
@@ -298,6 +298,75 @@ label mas_mood_bored:
...
+    m 2g "Oh my gosh [player], you're injured?!"
+    m 2n "I mean... on the bright side, you were able to make it here with me..."
+    m 2f "Is it really that bad [player]?"
+    menu:
+        "Just something minor":
+            m 2l "Ok! What a relief..."
+            m 1l "Don't scare me like that [player]!"
+            m 1e "I'm so glad you're alright but still..."
+            m 1f "I hope it really isn't too bad and you're not just trying not to worry me."
+            m 1l "Or maybe I'm just too worred about you~"
```
Comment (Babushka93) [inline Monika After Story/game/script-moods.rpy]: worried is missing the 'i' - Line 316

**Comment by Babushka93** [comment]
worried is missing the 'i' - Line 316

**Comment by ThePotatoGuy** [comment]
okay so fyi, we are going to merge this into a different branch. We're working on a better sick system, and this will be part of it.

**Comment by ThePotatoGuy** [comment]
[this has been merged into `better-sick` branch]

**Pull request closed without merging**